### PR TITLE
RMMIS-6076-terms-updates

### PR DESCRIPTION
**Pull request by kjschulz**
RMMIS-6076: Replaced containing div element with a fieldset and made a form-group, added an optional legend and checkbox header

### CODE REVIEW

File: src/Terms.jsx [hunk 1]
```diff
@@ -39,28 +42,46 @@ class Terms extends React.Component {
...
+        content={this.props.checkboxHeader} />
+    ) : (
+      null
+    );
+  }
+
   render() {
     return (
-      <div>
+      <fieldset className="form-group">
```
Comment (mallenscdhhs) [inline src/Terms.jsx]: You could use the `Fieldset` component which would contain all the necessary classNames, & such.

**Comment by mallenscdhhs** [comment]
@kjschulz do the unit tests need to be updated for this change?

File: src/Terms.jsx (hunk 1 above)
Comment (kjschulz) [inline src/Terms.jsx]: I don't think our `fieldset` component automatically uses the `field-group` class and I didn't want to include a couple other unnecessary elements (like the 2 `spans` in the legend ) that are added but I will if you prefer.

**Comment by kjschulz** [comment]
@mallenscdhhs, the unit tests do not need to be updated unless you want me to test for the presence of the optional params...otherwise the tests for the crucial functionality pass as is.

File: src/Terms.jsx (hunk 1 above)
Comment (mallenscdhhs) [inline src/Terms.jsx]: Fair enough.

**Pull request merged**